### ISSUE TITLE
grpc-js: ServerWritableStream.end(metadata?: Metadata) is missing from type definition

### ISSUE THREAD

**Issue by thomassuckow** [grpc/grpc-node#1520]
### Problem description
`ServerWritableStream.end(metadata?: Metadata)` is missing from type definition. This method is overriding the streams `end` method with different arguments. It works out if you don't include an argument because the stream has a variant like that but you can't send metadata without lying about the type.

### Reproduction steps
```typescript
const metadata = new Metadata();
stream.end(metadata as any);
```

### Environment
 - @grpc/grpc-js@1.1.1

**Comment by murgatroid99** [comment]
Thank you for reporting this. I have a fix in #1525.

**Comment by murgatroid99** [comment]
I have published the fix in version 1.1.4.

**Comment by thomassuckow** [comment]
![Not Worthy](https://media2.giphy.com/media/K55exy0toWjQc/giphy.gif)
